src/components/Login.js

```diff
@@ -99,31 +107,52 @@ export default function Login() {
     }
 
     return (
-      <Form onSubmit={submitOtp}>
-        <Form.Group size="lg">
-          <Form.Control
-            autoFocus
-            type="text"
-            placeholder="Enter Auth Code"
-            value={otp}
-            onChange={(e) => setOtp(e.target.value)}
-          />
-        </Form.Group>
-        <Button block size="lg" type="submit" disabled={!validAuthCode(otp)}>
-          Login
-        </Button>
-      </Form>
+      <form onSubmit={submitOtp}>
+        <input
+          id="otp"
+          autoFocus
+          required
+          type="text"
+          placeholder="Enter Auth Code"
+          value={otp}
+          onChange={(e) => setOtp(e.target.value)}
+          className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 focus:outline-none focus:shadow-outline"
+        />
+        <p className="text-red-500 mt-2 text-xs italic">
+          {otp && !validAuthCode(otp) ? "Invalid auth code" : ""}
+        </p>
+        <button
+          type="submit"
+          disabled={!validAuthCode(otp)}
```

Comment:
Similar to the above comment about ! in front of conditionals, could we change this to make it xxx===yyy?